DEVELOPERS.md

```diff
@@ -68,6 +68,18 @@
          `Console` tab to view printed logs and errors, the `Element` tab to
          inspect the HTML and CSS, the `Network` tab for network calls, etc).
 
+#### Debugging Add-in: Alternate way
```

Comment:
Suffix previous heading with "Method 1" so that user can expect an alternate way further on.

Reply:
@algomaster99 
Done